src/java/org/apache/cassandra/config/Config.java

```diff
@@ -1327,6 +1329,25 @@ public enum BatchlogEndpointStrategy
         }
     }
 
+    public enum TombstonesMetricGranularity
+    {
+        /**
+         * do not collect the metric at all
+         */
+        disabled,
+        /**
+         * track only partition/range/row level tombstone,
+         * a good compromise between overheads and usability
+         */
+        row,
+        /**
+         * track partition/range/row/cell level tombstones,
+         * the most granular option, but it has some performance overheads
```

Comment:
It would be nice to quantify what "some overheads" mean.  How much slower will it be? So a user knows what he is buying into.